molo/profiles/tests/test_admin.py

```diff
@@ -184,6 +183,26 @@ def test_staff_users_are_not_shown(self):
         self.assertContains(response, self.user.username)
         self.assertNotContains(response, self.superuser.email)
 
+    def test_staff_admin_are_shown(self):
+        user1 = User.objects.create_user(
+            username='testingstaff1', is_staff=True)
+        user2 = User.objects.create_user(
+            username='testingstaff2', is_staff=True)
+        user3 = User.objects.create_user(
+            username='testingstaff3', is_staff=True)
+
+        user1.profile.admin_sites.add(self.main.get_site())
+        user3.profile.site = None
+        user3.profile.save()
+
+        response = self.client.get(
+            '/admin/auth/user/?usertype=admin'
+        )
+        self.assertContains(response, user1.username)
+        self.assertContains(response, user2.username)
```

Comment:
Why does this user show up here? the site hasn't been added to their `admin_sites`

Reply:
@KaitCrawford The superuser has a staff status, see the create_superuser User's method definition

Reply:
but this isn't a superuser? This is user2 that I'm asking about. You don't set their `admin_sites` anywhere but then why are they showing up in the user list?
But I see you had to manually set user3's `profile.site` to none. Do users have a site set by default?

Reply:
@KaitCrawford please see above comments and and filter code to answer your questions